### DISCUSSION TITLE
Is there a plan to support Http 3?

### DISCUSSION THREAD

**Discussion by HaoYang670** [tokio-rs/axum#2636] [Q&A] [answered]
### Summary

As the title

### axum version

main HEAD

**Comment by jplatte** [comment, marked as answer]
axum does not implement any version of the HTTP protocol itself, it just implements routing, request handling and a bunch of convenience things on top of that. `axum::serve` is a very thin abstraction on top of the `hyper` crate (and its sibling crates), and once hyper supports HTTP/3, axum will automatically support it as well.

This is the issue you should subscribe to to get news about HTTP/3 support in hyper: https://github.com/hyperium/hyper/issues/1818
